include/aikido/util/string.hpp

```diff
@@ -0,0 +1,30 @@
+#ifndef AIKIDO_UTIL_STRING_HPP_
+#define AIKIDO_UTIL_STRING_HPP_
+
+#include <vector>
+#include <string>
+
+namespace aikido {
+namespace util {
+
+/// Splits (tokenizes) a string into substrings that are divided by the given
+/// delimiters.
+///
+/// Example:
+/// \code
+/// std::string text = "HERB, the robot.";
+///
+/// auto res1 = split(text); // {"HERB,", "the", "robot."}
+/// auto res2 = split(text, ","); // {"HERB", " the robot."}
+/// \endcode
+///
+/// \param[in] string Input string to be splitted.
+/// \param[in] delimiters Delimiters.
```

Comment:
Add a note that *any* character this string is considered a delimiter, i.e. the string is interpreted as a set of delimiter characters, not as a single multi-character delimiter.